package.json

```diff
@@ -5,14 +5,13 @@
   "source": "src/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.module.js",
-  "esmodule": "dist/index.esmodule.js",
```

Comment:
This was suggested by Microbundle but has been replaced entirely with `"exports"` as of Node 14 I believe. I think a few tools support it, but it's not worth having really.

We could (should?) make the `"browser"` key the consumer of the "modern" output.

Reply:
Not sure about that 🤔 Anyway, we have to postpone that's until the next major release.

Reply:
Any build tool would fall back to plain ESM, but you're right, I suppose if someone's using a direct URL via a CDN or something then that's no good. My bad. Will revert.